File: sources/im/getsocial/sdk/usermanagement/UsersQuery.java
package im.getsocial.sdk.usermanagement;

public final class UsersQuery {
    /* renamed from: a */
    private final String f3269a;
    /* renamed from: b */
    private int f3270b = 20;

    private UsersQuery(String str) {
        this.f3269a = str;
    }

    public static UsersQuery usersByDisplayName(String str) {
        return new UsersQuery(str);
    }

    public final int getLimit() {
        return this.f3270b;
    }

    public final String getQuery() {
        return this.f3269a;
    }

    public final UsersQuery withLimit(int i) {
        this.f3270b = i;
        return this;
    }
}

File: assets/bin/Data/Managed/Assembly-CSharp/GetSocialSdk.Core/UsersQuery.cs
using UnityEngine;

namespace GetSocialSdk.Core
{
	public class UsersQuery : IConvertableToNative
	{
		private const int DefaultLimit = 20;

		private readonly string _query;

		private int _limit;

		private UsersQuery(string query)
		{
			_query = query;
			_limit = 20;
		}

		public static UsersQuery UsersByDisplayName(string query)
		{
			return new UsersQuery(query);
		}

		public UsersQuery WithLimit(int limit)
		{
			_limit = limit;
			return this;
		}

		public AndroidJavaObject ToAjo()
		{
			//IL_0005: Unknown result type (might be due to invalid IL or missing references)
			//IL_001e: Expected O, but got Unknown
			return new AndroidJavaClass("im.getsocial.sdk.usermanagement.UsersQuery").CallStaticAJO("usersByDisplayName", _query).CallStaticAJO("withLimit", _limit);
		}
	}
}
